README.md

```diff
@@ -27,10 +34,299 @@ To build all apps and packages, run the following command:
 pnpm run build
 ```
 
+You can also build a specific app or package by running this command:
+
+```bash
+pnpm dlx turbo run build --filter=checkout
+# or
+pnpm run build:checkout
+```
+
+In this example, we'll only build `apps/checkout`
+
 ### Develop
 
-To develop all apps and packages, run the following command:
+Create a tunnel for `checkout-app`:
+
+```bash
+cd apps/checkout-app && npx saleor app tunnel 3000
+```
+
+> Note: the process needs to be running in the background
+
+Before you start the server, you need to change default environment variables. Create `.env.local` file in each app:
+- [`apps/checkout-app`](./apps/checkout-app/README.md#env-variables)
+- [`apps/checkout`](./apps/checkout/README.md#local-development)
+
+To run the development server for all the apps, use the following command:
 
 ```
 pnpm run dev
 ```
+
+You can also run only a specific app by running this command:
+
+```bash
+pnpm dlx turbo run dev --filter=checkout
+# or
+cd apps/checkout && pnpm dev
```

Comment:
Please stick with one command. BTW: Why the first one does not have `cd ...`?

Reply:
As in prev one we can remove the first one, as it's more verbose and we don't have more than 2 apps.

The reason for `cd` is that `pnpm` will run scripts from the closest `package.json` so if you're in `apps/checkout` directory it will run the `dev` script defined in `apps/checkout/package.json`